packages/lesswrong/server/autocompleteEndpoint.ts

```diff
@@ -130,12 +141,22 @@ async function constructMessageHistory(
 
   if (replyingCommentId) {
     // Fetch the comment we're replying to
-    const replyingToCommentResponse = await runFragmentQuery({
-      collectionName: "Comments",
-      fragmentName: "CommentsForAutocompleteWithParents",
-      terms: { commentIds: [replyingCommentId] },
-      context,
-    })
+    const replyingToCommentResponse = await getWithCustomLoader(context, "commentsWithParentsForLLM", replyingCommentId, async (commentIds: string[]) => {
+      const comments = await runFragmentQuery({
+        collectionName: "Comments",
+        fragmentName: "CommentsForAutocompleteWithParents",
+        terms: { commentIds: commentIds },
+        context,
+      })
+      const upwardCommentClosure = (commentId: string): CommentsForAutocompleteWithParents[] => {
```

Comment:
If we're no longer relying on the fragment to get the parent comment tree, has the fragment itself been cleaned up?